README.md

```diff
@@ -21,46 +21,54 @@ bundle
 bundle exec rails g solidus_paypal_braintree:install
 ```
 
-Usage
------
+## Basic Setup
 
-This gem extends Solidus by providing a new payment method and source, named
-`SolidusPaypalBraintree::Gateway` and `SolidusPaypalBraintree::Source` respectively.
-All payment types - PayPal, ApplePay, and Credit Cards - are supported through
-the same payment method.
+### Retrieve Braintree account details
+You'll need the following account details:
+- `Merchant ID`
+- `Public key`
+- `Private key`
 
-The payment method requires 3 preferences to be set to process payments:
-- `merchant_id`
-- `public_key`
-- `private_key`
-
-These values can be obtained by logging in to your Braintree account and going
+These values can be obtained by logging in to your Braintree account, going
 to `Account -> My User` and clicking `View Authorizations` in the **API Keys,
 Tokenization Keys, Encryption Keys** section.
 
-The payment method also provides an optional preference `merchant_currency_map`.
-This preference allows users to provide different Merchant Account Ids for
-different currencies. If you only plan to accept payment in one currency, the
-defaut Merchant Account Id will be used and you can omit this option.
-An example of setting this preference can be found
-[here](https://github.com/solidusio/solidus_paypal_braintree/blob/master/spec/spec_helper.rb#L70-L72).
+### Create a new payment method
+1. Visit `/admin/payment_methods/new`.
+2. Set `provider` to SolidusPaypalBraintree::Gateway
+3. Enter the three required preferences: `merchant_id`, `public_key`, `private_key`. Other optional preferences are discussed below.
+4. Click `Update` to save
 
-Store Configuration
--------------------
+Alternatively, create a payment method from the Rails console with:
+```ruby
+SolidusPaypalBraintree::Gateway.new(
+  name: "Braintree",
+  preferences: {
+    public_key:  '[your public key]',
+    private_key: '[your private key]',
+    merchant_id: '[your merchant ID]'
+  }
+).save
+```
```

Comment:
OK, makes sense. I think we need to cover both approaches, since they're both provided. But I'll write it to indicate that static preferences are preferred.